crates/turborepo-lib/src/commands/unlink.rs

```diff
@@ -11,26 +11,48 @@ enum UnlinkSpacesResult {
 }
 
 fn unlink_remote_caching(base: &mut CommandBase) -> Result<()> {
-    base.delete_repo_config_file()
-        .context("could not unlink. Something went wrong")?;
+    let repo_config: &mut crate::config::RepoConfig = base.repo_config_mut()?;
+    let needs_disabling = repo_config.team_id().is_some() || repo_config.team_slug().is_some();
```

Comment:
Would it be worth moving this logic into the config struct itself to something like `maybe_disable_remote_caching -> Result<bool, Error>`?

Reply:
I _almost_ did this, but I want to remove `team_slug` entirely and change the shape of how this works by default. It felt premature.